pkg/odo/cli/url/create.go

```diff
@@ -344,6 +448,11 @@ func NewCmdURLCreate(name, fullName string) *cobra.Command {
 			urlCreateCmd.Flags().StringVar(&o.tlsSecret, "tls-secret", "", "TLS secret name for the url of the component if the user bring their own TLS secret")
 			urlCreateCmd.Flags().StringVarP(&o.host, "host", "", "", "Cluster IP for this URL")
 			urlCreateCmd.Flags().BoolVar(&o.wantIngress, "ingress", false, "Create an Ingress instead of Route on OpenShift clusters")
+			urlCreateCmd.Flags().BoolVarP(&o.secureURL, "secure", "", false, "Create a secure URL")
+			urlCreateCmd.Flags().StringVarP(&o.exposure, "exposure", "", "", "URL exposure")
```

Comment:
I feel we don't need to provide a exposure flag option to the use and instead make all URLs public.

Reply:
Makes sense.

Reply:
updated